app/views/docs/getting-started-for-server.phtml

```diff
@@ -183,6 +226,15 @@ Response user = await account
     .createJWT();</code></pre>
         </div>
     </li>
+    <li>
+        <h3>Android SDK</h3>
```

Comment:
I don't think this needs to be in getting started for server.

Reply:
this is client side creating JWT token. not server side.

Reply:
Ah okay :+1: